### PR TITLE
Update HA settings in helm chart

### PR DESCRIPTION
**Pull request by abowloflrf**
Signed-off-by: Ruofeng Lei <ruofenglei@outlook.com>

### Ⅰ. Describe what this PR does

For #1960 

1. add replicas setting in helm chart values, default to 1 for each component
2. enable leader election for all components by default
3. fix RBAC issue for leader election 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/fluid/fluid/templates/controller/alluxioruntime_controller.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ spec:
           - --runtime-node-port-range={{ .Values.runtime.alluxio.portRange }}
           - --runtime-workers={{ .Values.runtime.alluxio.runtimeWorkers }}
           - --pprof-addr=:6060
+          - --enable-leader-election
```
Comment (cheyang) [inline charts/fluid/fluid/templates/controller/alluxioruntime_controller.yaml:42]: I think if the replicas is  1, the `--enable-leader-election` is disabled. In other cases,  it's enabled.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: charts/fluid/fluid/templates/controller/alluxioruntime_controller.yaml (hunk 1 above)
Comment (abowloflrf) [reply charts/fluid/fluid/templates/controller/alluxioruntime_controller.yaml:42]: Let the leader election enable in single replicas case is fine I think, the controller also works well.

What's more, it also prevents some unexpected errors when user install with `replicas=1` first and then scale deployment to >1 after that.

Comment (cheyang) [reply charts/fluid/fluid/templates/controller/alluxioruntime_controller.yaml:42]: I've tried the latest change, but got the following error

```bash
logs -n fluid-system jindoruntime-controller-88c78bc8c-k8j86
2022/07/24 16:09:27 BuildDate: 2022-07-24_07:38:29
2022/07/24 16:09:27 GitCommit: c7f588812aeaa4e2b05271bd1e9ffc431555264b
2022/07/24 16:09:27 GitTreeState: clean
2022/07/24 16:09:27 GoVersion: go1.17.8
2022/07/24 16:09:27 Compiler: gc
2022/07/24 16:09:27 Platform: linux/amd64
2022-07-24T16:09:27.923+0800	INFO	setup	utils/pprof.go:32	Enabling pprof	{"pprof address": ":6060 - --enable-leader-election"}
2022-07-24T16:09:27.923+0800	INFO	setup	utils/pprof.go:47	Starting pprof HTTP server	{"pprof server address": ":6060 - --enable-leader-election"}
2022-07-24T16:09:28.019+0800	ERROR	setup	utils/pprof.go:63	Failed to start debug HTTP server	{"error": "listen tcp: address tcp/6060 - --enable-leader-election: unknown port"}
github.com/fluid-cloudnative/fluid/pkg/utils.NewPprofServer.func1
	/go/src/github.com/fluid-cloudnative/fluid/pkg/utils/pprof.go:63
panic: listen tcp: address tcp/6060 - --enable-leader-election: unknown port

goroutine 32 [running]:
github.com/fluid-cloudnative/fluid/pkg/utils.NewPprofServer.func1()
	/go/src/github.com/fluid-cloudnative/fluid/pkg/utils/pprof.go:64 +0x1c5
created by github.com/fluid-cloudnative/fluid/pkg/utils.NewPprofServer
	/go/src/github.com/fluid-cloudnative/fluid/pkg/utils/pprof.go:49 +0x516
```

File: charts/fluid/fluid/templates/controller/jindoruntime_controller.yaml [hunk 2]
```diff
@@ -38,6 +38,7 @@ spec:
             - --runtime-node-port-range={{ .Values.runtime.jindo.portRange }}
             - --runtime-workers={{ .Values.runtime.jindo.runtimeWorkers }}
             - --pprof-addr=:6060
+             - --enable-leader-election
```
Comment (cheyang) [inline charts/fluid/fluid/templates/controller/jindoruntime_controller.yaml:41]: The indentation of `--enable-leader-election` doesn't match the line above.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: charts/fluid/fluid/templates/controller/jindoruntime_controller.yaml (hunk 2 above)
Comment (abowloflrf) [reply charts/fluid/fluid/templates/controller/jindoruntime_controller.yaml]: my fault, fixed 😬

**Review by cheyang — APPROVED** [review]

**Comment by cheyang** [comment]
Thanks very much for your contribution! @abowloflrf  If it's possible, please add document about how to use it. Thanks again.

**Pull request merged**